_posts/2022-08-25-cockpit-275.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Cockpit 275
+author: jelle
+date: '2022-08-25'
+tags: cockpit, podman
+slug: cockpit-275
+category: release
+summary: 'Support for alternatives to sudo, image history, restart policy, container ports, volumes and env's '
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 275 and cockpit-podman 52:
+
+
+## Shell: Support for alternatives to sudo
+
+Cockpit has always the theoretical ability to run arbitrary programs to gain administrative access, by adding them to a package manifest.  However, the UI would always use "sudo". Now, if multiple programs are defined, the UI will prompt for which one to use.
```

Comment:
"Cockpit always had the theoretical ability"?

Reply:
*has always had, i'd say

Reply:
(don't ask me to explain this.  english is weird.)

Reply:
Applied!